oak_remote_attestation_noninteractive/proto/v1/messages.proto

```diff
@@ -24,10 +24,23 @@ option java_package = "oak.session.noninteractive.v1";
 message PublicKeyInfo {
   // The serialized enclave public key.
   // TODO(#3442): Specify key format.
-  bytes public_key = 1;
+  bytes enclave_public_key = 1;
```

Comment:
I think we need an additional field to specify the intended use of the public key. It is usually important to know whether a key is used for signing or encryption. Perhaps we should always create two keys, one signing key and one encryption key.

Reply:
My understanding from the description in https://github.com/project-oak/oak/blob/main/docs/remote-attestation.md is that this key is for encryption. Are there any scenarios where the key would be used for signing?

Reply:
If we want to sign some future content to prove that it came from the TEE we need a signing key. An example would be the future plans to implement an interactive handshake with an attestation verification service that will allow the TEE to prove its identity. The TEE would generate an ephemeral key pair as part of the handshake and then must be able to prove to the verification service that the ephemeral key pair was actually generated in the TEE.